mopidy/core/playback.py

```diff
@@ -479,6 +478,8 @@ def stop(self):
         if self.get_state() != PlaybackState.STOPPED:
             self._last_position = self.get_time_position()
             backend = self._get_backend(self.get_current_tl_track())
+            if not backend and self._pending_tl_track is not None:
+                backend = self._get_backend(self._pending_tl_track)
```

Comment:
One way to provoke this is to start playback on a backend that is slow to start (eg Youtube) and then remove the track from the tracklist before it starts playing. Somehing that happens to me in real-life usage on a regular basis.  Given that this code is in .stop() then you always want playback to stop so by definition it's safe to stop the pending track if there's no current track. There are other ways to provoke it but that's the easiest one to explain. Since I put that change in my local copy I've had no occurrences of there being a track playing that isn't in the tracklist, which used to happen to me several times a day.